drake/systems/plants/test/testSignedDistance.m

```diff
@@ -0,0 +1,49 @@
+function testSignedDistance
+
+visualize = 0;
+options.floating = true;
+options.terrain = RigidBodyFlatTerrain();
+
+s = 'FallingBrick.urdf';
```

Comment:
Running it on my own machine shows it to work OK with zero-rad spheres -- it looks like it gets signed (in this case nonnegative) distance and closest body correct without issue.

I'll update the test to include this (and I'll throw in some more careful checking of the returned signed distances).